recipes/shorah/1.99.1/conda_build_config.yaml

```diff
@@ -0,0 +1,2 @@
+blas:
+  - 1.1 openblas
```

Comment:
@DrYak wait sorry. I don't think this is needed or is working. Can you remove it?

Reply:
Pushed.

Reply:
Working: actually it does still work.
Needed: actually the whole BLAS isn't needed anymore since we moved from GSL to Boost.